src/main/java/io/github/fvarrui/javapackager/gradle/DefaultPackageTask.java

```diff
@@ -9,57 +9,59 @@
-					.assetsDir(extension.getAssetsDir())
-					.bundleJre(extension.getBundleJre())
-					.classpath(extension.getClasspath())					
-					.copyDependencies(extension.getCopyDependencies())
-					.createTarball(extension.getCreateTarball())
-					.createZipball(extension.getCreateZipball())
-					.customizedJre(extension.getCustomizedJre())
-					.description(extension.getDescription())
-					.displayName(extension.getDisplayName())
-					.envPath(extension.getEnvPath())
-					.extra(extension.getExtra())
-					.fileAssociations(extension.getFileAssociations())
-					.forceInstaller(extension.isForceInstaller())
-					.generateInstaller(extension.getGenerateInstaller())
-					.jdkPath(extension.getJdkPath())
-					.jreDirectoryName(extension.getJreDirectoryName())
-					.jreMinVersion(extension.getJreMinVersion())
-					.jrePath(extension.getJrePath())
-					.licenseFile(extension.getLicenseFile())
-					.linuxConfig(extension.getLinuxConfig())
-					.macConfig(extension.getMacConfig())
-					.mainClass(extension.getMainClass())
-					.manifest(extension.getManifest())
-					.modules(extension.getModules())
-					.name(extension.getName())
-					.organizationEmail(extension.getOrganizationEmail())
-					.organizationName(extension.getOrganizationName())
-					.organizationUrl(extension.getOrganizationUrl())
-					.outputDirectory(extension.getOutputDirectory())
-					.packagingJdk(extension.getPackagingJdk())
-					.runnableJar(extension.getRunnableJar())
-					.scripts(extension.getScripts())
-					.useResourcesAsWorkingDir(extension.isUseResourcesAsWorkingDir())
-					.url(extension.getUrl())
-					.version(defaultIfNull(extension.getVersion(), getProject().getVersion().toString()))
-					.vmArgs(extension.getVmArgs())
-					.winConfig(extension.getWinConfig());
+
+		return
+				(Packager) PackagerFactory
+						.createPackager(extension.getPlatform())
+						.additionalModules(extension.getAdditionalModules())
+						.additionalModulePaths(extension.getAdditionalModulePaths())
+						.additionalResources(extension.getAdditionalResources())
+						.administratorRequired(extension.getAdministratorRequired())
+						.assetsDir(extension.getAssetsDir())
+						.bundleJre(extension.getBundleJre())
+						.classpath(extension.getClasspath())
+						.copyDependencies(extension.getCopyDependencies())
+						.createTarball(extension.getCreateTarball())
+						.createZipball(extension.getCreateZipball())
+						.customizedJre(extension.getCustomizedJre())
+						.description(extension.getDescription())
+						.displayName(extension.getDisplayName())
+						.envPath(extension.getEnvPath())
+						.extra(extension.getExtra())
+						.fileAssociations(extension.getFileAssociations())
+						.forceInstaller(extension.isForceInstaller())
+						.generateInstaller(extension.getGenerateInstaller())
+						.jdkPath(extension.getJdkPath())
+						.jdkVersion(extension.getJdkVersion())
+						.jdkVendor(extension.getJdkVendor())
```

Comment:
Idk why this file has so many diffs, maybe because it was completely different before, but
the only thing changed are the two lines above.

Reply:
No idea 😅 ... Git diff algorythm? Your IDE replacing tabs with whitespaces or new lines style?